nucypher/blockchain/eth/agents.py

```diff
@@ -102,6 +110,12 @@ class EthereumContractAgent:
     class ContractNotDeployed(Exception):
         pass
 
+    class RequirementError(Exception):
```

Comment:
:tada:

Reply:
Yeah, from this exception, we can start to provide better error handling on the CLI layer.